client/packages/system/src/Item/Components/ItemVariant/ItemVariantPackUnit.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
```

Comment:
This cell is used in stock view now, to show unit variant short name (based on pack size) instead of pack size and unit column